packages/tux-adapter-contentful/src/query-api.ts

```diff
@@ -108,21 +107,20 @@ class QueryApi {
 
     if (isLeaf) {
       if (item.sys && item.sys.type === 'Link') {
-        const linkType = item.sys.linkType.toLowerCase()
-        item[linkType] =
-          linkMap[item.sys.id] ||
-          (this.overrides[item.sys.id] && this.overrides[item.sys.id].fields)
-
-        // Link nested models
-        const fieldNames = Object.keys(item[linkType])
-        fieldNames.forEach(fieldName => this.linkFields(item[linkType][fieldName], linkMap))
+        const entry = linkMap[item.sys.id] || (this.overrides[item.sys.id] && this.overrides[item.sys.id].fields)
```

Comment:
This can be simplified to `const entry = linkMap[item.sys.id] || this.overrides[item.sys.id]` since we want the whole entry.